### ISSUE TITLE
[OTHER] Background's contrasting colors are not accessible enough 

### ISSUE THREAD

**Issue by CBID2** [rupali-codes/LinksHub#1045]
### What would you like to share or ask?

I just ran an accessibility check with [this tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related). Apparently, the contrasting colors in the website's background are not up to WCAG standards(see screenshot below). 

![accessibility test results](https://github.com/rupali-codes/LinksHub/assets/105683440/8b6b9992-ce65-4f4d-8605-4bf0e649eab8)
While I love the colors, I don't want developers/tech workers who are vision-impaired to have trouble with navigating LinksHub, so maybe there can be other color changes. What do you think @rupali-codes? Feel free to share your thoughts when you're free @ujjawaltyagii and @Anmol-Baranwal.   

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [x] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

**Comment by rupali-codes** [comment]
@CBID2 Are you talking about to change the whole color?

**Comment by CBID2** [comment]
> @CBID2 Are you talking about to change the whole color?

Yes @rupali-codes

**Comment by ujjawaltyagii** [comment]
Are we going the change the entire theme of website or just some variation of current color 0_0 ??

**Comment by CBID2** [comment]
> @CBID2  Are we going the change the entire theme of website or just some variation of current color 0_0 ??

I'm learning towards the latter @ujjawaltyagii. Like pick colors that are similar to what we currently have yet accessible.

**Comment by Anmol-Baranwal** [comment]
Sorry for the late response, as I have just finished my exam.

During my UX journey, I learned about the concept of "Next Billion Users," which is particularly emphasized by Google. It refers to the concept of expanding technology access and usability to the next billion individuals who are coming online from diverse backgrounds and may have unique needs and limitations. Google's focus on increasing accessibility for this user group is commendable.

I strongly support the idea of improving accessibility. However, it's important to acknowledge that even on top platforms like Amazon or any other, there will always be accessibility issues, ranging from basic to complex. While continuous improvements are made to ensure applications are accessible, it's crucial not to solely base an application's design on accessibility. Instead, we should strive to make the application as accessible as possible, incorporating maximum accessibility features. For example, even if we enhance contrast, there will still be challenges.

Ultimately, the balance between aesthetics and accessibility depends on individual preferences. Completely changing colors to improve contrast can have both positive and negative impacts (almost nada). Personally, I wouldn't recommend making such changes unless they significantly affect real users. In this particular case, the impact might not be substantial.

I hope you understand that I am simply expressing my thoughts, which may not be entirely accurate. Please feel free to share your own perspective, and I appreciate @CBID2 for considering accessibility, as it is often overlooked by many :)

**Comment by CBID2** [comment]
Thank you for your insight @Anmol-Baranwal! :) Personally, I think this is something we should really do. While the impact might not be a big deal to us, it probably would be for the vision-impaired person. I mean sure there might be drawbacks but as long as the change creates a more comfortable experience, then it's worth it.

**Comment by Anmol-Baranwal** [comment]
> Thank you for your insight @Anmol-Baranwal! :) Personally, I think this is something we should really do. While the impact might not be a big deal to us, it probably would be for the vision-impaired person. I mean sure there might be drawbacks but as long as the change creates a more comfortable experience, then it's worth it.

I totally agree 👍

**Comment by EmmaDawsonDev** [comment]
> @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea. 

Thanks ☺️ 

In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker. 

This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.

**Comment by CBID2** [comment]
> > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea. 
> 
> Thanks ☺️ 
> 
> In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker. 
> 
> This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.

Thanks @EmmaDawsonDev! :)

**Comment by Anmol-Baranwal** [comment]
> > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea.
> 
> Thanks ☺️
> 
> In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker.
> 
> This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.

That's a precise level suggestion.

@CBID2, when are we adding this change?

**Comment by rupali-codes** [comment]
> > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea.
> 
> Thanks ☺️
> 
> In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker.
> 
> This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.

Thanks @EmmaDawsonDev, we can for it :)

**Comment by rupali-codes** [comment]
> > > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea.
> > 
> > 
> > Thanks ☺️
> > In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker.
> > This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.
> 
> That's a precise level suggestion.
> 
> @CBID2, when are we adding this change?

let's look up the colors then we can go for it :)

**Comment by CBID2** [comment]
> > > > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea.
> 
> > > 
> 
> > > 
> 
> > > Thanks ☺️
> 
> > > In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker.
> 
> > > This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.
> 
> > 
> 
> > That's a precise level suggestion.
> 
> > 
> 
> > @CBID2, when are we adding this change?
> 
> 
> 
> let's look up the colors then we can go for it :)

There's your answer @Anmol-Baranwal

**Comment by CBID2** [comment]
> > > @EmmaDawsonDev, as the digital queen of accessibility (lol that is my nickname for you 😄😉), I'd like to hear your opinion on my idea.
> > 
> > 
> > Thanks ☺️
> > In this case its a case of bumping the contrast up to at least 4.5:1 which will mean new colours chosen would be fairly similar. To fix this you could either make the purple lighter or the background darker.
> > This is an easy accessibility fix, won't have much impact on the design and although it might not make a difference to lots of people it shows you care and make an effort towards being more accessible and more inclusive.
> 
> Thanks @EmmaDawsonDev, we can for it :)

Hey @rupali-codes, I found the new accessible versions of our current colors:
![new colors](https://github.com/rupali-codes/LinksHub/assets/105683440/446a1eab-ea95-49e9-b68e-357d32594812)
What do you think? @EmmaDawsonDev, I'd love your input on this too. Same as you @Anmol-Baranwal and @ujjawaltyagii! :)

**Comment by rupali-codes** [comment]
@CBID2 looks good though

**Comment by CBID2** [comment]
> @CBID2 looks good though

Great! :)

**Comment by EmmaDawsonDev** [comment]
Might just be a good idea to check the new colors work with light mode too, if any like the purple are used in both

**Comment by CBID2** [comment]
> Might just be a good idea to check the new colors work with light mode too, if any like the purple are used in both

Ooh good point @EmmaDawsonDev

**Comment by k-deepak04** [comment]
@CBID2 working on this?

**Comment by CBID2** [comment]
> @CBID2 working on this?

Yeah I just need to figure out where to make the changes file-wise @k-deepak04z.

**Comment by k-deepak04** [comment]
> > @CBID2 working on this?
> 
> Yeah I just need to figure out where to make the changes file-wise @k-deepak04z.

okay madam :)

**Comment by CBID2** [comment]
> > > @CBID2 working on this?
> 
> > 
> 
> > Yeah I just need to figure out where to make the changes file-wise @k-deepak04z.
> 
> 
> 
> okay madam :)

@rupali-codes, can you point out where I can make these changes?

**Comment by rupali-codes** [comment]
I'm thinking to configure all the colors in tailwing.config file, it would be so helpful for us. then you can change it directly. So, should i work on that, bcz its gonna be a bit difficult for you to find where the components are located. @k-deepak04 or @CBID2 you guys can create an issue for it, and assign to me, i'll work on that :)

**Comment by CBID2** [comment]
> I'm thinking to configure all the colors in tailwing.config file, it would be so helpful for us. then you can change it directly. So, should i work on that, bcz its gonna be a bit difficult for you to find where the components are located. @k-deepak04 or @CBID2 you guys can create an issue for it, and assign to me, i'll work on that :)

Sure @rupali-codes! :)

**Comment by rupali-codes** [comment]
> > I'm thinking to configure all the colors in tailwing.config file, it would be so helpful for us. then you can change it directly. So, should i work on that, bcz its gonna be a bit difficult for you to find where the components are located. @k-deepak04 or @CBID2 you guys can create an issue for it, and assign to me, i'll work on that :)
> 
> Sure @rupali-codes! :) To save time, how about I just assign you to this one?

i guess making another issue would be better, you can create a new issue and assign that to me ;)

**Comment by CBID2** [comment]
Ok

**Comment by CBID2** [comment]
Here it is: https://github.com/rupali-codes/LinksHub/issues/1372

**Comment by rupali-codes** [comment]
> Here it is @rupali-codes: #1372

Hi Chrissy, are you making sperate PR for it?

**Comment by CBID2** [comment]
> > Here it is @rupali-codes: #1372
> 
> Hi Chrissy, are you making sperate PR for it?

I thought you were going to @rupali-codes lol

**Comment by rupali-codes** [comment]
> > > Here it is @rupali-codes: #1372
> > 
> > 
> > Hi Chrissy, are you making sperate PR for it?
> 
> I thought you were going to @rupali-codes lol

I didn't have the colors you chose,

**Comment by CBID2** [comment]
> > > > Here it is @rupali-codes: #1372
> 
> > > 
> 
> > > 
> 
> > > Hi Chrissy, are you making sperate PR for it?
> 
> > 
> 
> > I thought you were going to @rupali-codes lol
> 
> 
> 
> I didn't have the colors you chose, 

Ohh ok. Then I'll add them
